DQM/L1TMonitor/python/L1TStage2Emulator_cff.py

```diff
@@ -7,89 +7,102 @@
+    ecalToken = "caloLayer1Digis",
+    hcalToken = "caloLayer1Digis",
+    unpackEcalMask = True,
+    unpackHcalMask = True
+)
 
 # CaloLayer2
 from L1Trigger.L1TCalorimeter.simCaloStage2Digis_cfi import simCaloStage2Digis
-valCaloStage2Layer2Digis = simCaloStage2Digis.clone()
-valCaloStage2Layer2Digis.towerToken = cms.InputTag("caloStage2Digis", "CaloTower")
+valCaloStage2Layer2Digis = simCaloStage2Digis.clone(towerToken = "caloStage2Digis:CaloTower")
 
 # BMTF-Legacy
 from L1Trigger.L1TMuonBarrel.simBmtfDigis_cfi import *
-valBmtfDigis = simBmtfDigis.clone()
-valBmtfDigis.DTDigi_Source = cms.InputTag("bmtfDigis")
-valBmtfDigis.DTDigi_Theta_Source = cms.InputTag("bmtfDigis")
+valBmtfDigis = simBmtfDigis.clone(
+    DTDigi_Source = "bmtfDigis",
+    DTDigi_Theta_Source = "bmtfDigis"
+)
 
 # BMTF-Kalman
 from L1Trigger.L1TMuonBarrel.simKBmtfDigis_cfi import *
 from L1Trigger.L1TMuonBarrel.simKBmtfStubs_cfi import *
-valKBmtfStubs = simKBmtfStubs.clone()
-valKBmtfStubs.srcPhi = cms.InputTag("bmtfDigis")
-valKBmtfStubs.srcTheta = cms.InputTag("bmtfDigis")
-valKBmtfDigis = simKBmtfDigis.clone()
-valKBmtfDigis.src = cms.InputTag("valKBmtfStubs")
+valKBmtfStubs = simKBmtfStubs.clone(
+    srcPhi = "bmtfDigis",
+    srcTheta = "bmtfDigis"
+)
+valKBmtfDigis = simKBmtfDigis.clone(src = "valKBmtfStubs")
 
 # BMTF-AlgoTriggerSelector
 from DQM.L1TMonitor.L1TBMTFAlgoSelector_cfi import *
-valBmtfAlgoSel = l1tBmtfAlgoSelector.clone()
-valBmtfAlgoSel.bmtfKalman = cms.InputTag("valKBmtfDigis:BMTF")
-valBmtfAlgoSel.bmtfLegacy = cms.InputTag("valBmtfDigis:BMTF")
+valBmtfAlgoSel = l1tBmtfAlgoSelector.clone(
+    bmtfKalman = "valKBmtfDigis:BMTF",
+    bmtfLegacy = "valBmtfDigis:BMTF"
+)
 
 # OMTF
 from L1Trigger.L1TMuonOverlap.simOmtfDigis_cfi import *
-valOmtfDigis = simOmtfDigis.clone()
-valOmtfDigis.srcDTPh = cms.InputTag('omtfStage2Digis')
-valOmtfDigis.srcDTTh = cms.InputTag('omtfStage2Digis')
-valOmtfDigis.srcCSC = cms.InputTag('omtfStage2Digis')
-valOmtfDigis.srcRPC = cms.InputTag('omtfStage2Digis')
+valOmtfDigis = simOmtfDigis.clone(
+    srcDTPh = "omtfStage2Digis",
+    srcDTTh = "omtfStage2Digis",
+    srcCSC = "omtfStage2Digis",
+    srcRPC = "omtfStage2Digis"
+)
 
 # GEM TPG
 from L1Trigger.L1TGEM.simGEMDigis_cff import *
-valMuonGEMPadDigis = simMuonGEMPadDigis.clone()
-valMuonGEMPadDigis.InputCollection = cms.InputTag('muonGEMDigis')
-valMuonGEMPadDigiClusters = simMuonGEMPadDigiClusters.clone()
-valMuonGEMPadDigiClusters.InputCollection = cms.InputTag('valMuonGEMPadDigis')
+valMuonGEMPadDigis = simMuonGEMPadDigis.clone(InputCollection = "muonGEMDigis")
+valMuonGEMPadDigiClusters = simMuonGEMPadDigiClusters.clone(InputCollection = "valMuonGEMPadDigis")
+
+# CSC TPG
+from L1Trigger.CSCTriggerPrimitives.cscTriggerPrimitiveDigis_cfi import *
+valCscStage2Digis = cscTriggerPrimitiveDigis.clone(
+    CSCComparatorDigiProducer = "muonCSCDigis:MuonCSCComparatorDigi",
+    CSCWireDigiProducer = "muonCSCDigis:MuonCSCWireDigi"
```

Comment:
In #33120 there are also
```
    GEMPadDigiClusterProducer = "",
    commonParam = dict(runME11ILT = False)
```

Reply:
@silviodonato See this comment https://github.com/cms-sw/cmssw/pull/33120#issuecomment-812543937 for the explanation.